README.md

```diff
@@ -88,28 +88,24 @@ badge build passed :green .png > mybadge.png
 ```
 
 ```js
-const badge = require('gh-badges')
+const { BadgeFactory } = require('gh-badges')
+
+const bf = new BadgeFactory({ fontPath: '/path/to/Verdana.ttf' })
 
-// Optional step, to have accurate text width computation.
 const format = {
   text: ['build', 'passed'],
   colorscheme: 'green',
   template: 'flat',
 }
 
-badge.loadFont('/path/to/Verdana.ttf', err => {
-  badge(format, (svg, err) => {
-    // svg is a string containing your badge
-  })})
+const svg = bf.create(format)
```

Comment:
So this is sync now?

Reply:
Yep. As noted in https://github.com/badges/shields/issues/1388#issuecomment-354929354 its now synchronous